apps/application-system/README.md

```diff
@@ -10,6 +10,18 @@ This project forms the base for all business applications belonging to island.is
 - [Staging](https://beta.staging01.devland.is/umsoknir/)
 - [Production](https://island.is/umsoknir/)
 
+
+## Quickstart
```

Comment:
following up on similar notes like @eirikurn I would suggest that the documentation in the quickstart section would indicate in more detail that they are a bundle of commands which are also outlined in the readme

Reply:
Updated. How do you like it now?